Docs/Performance_Metrics.md

```diff
@@ -0,0 +1,68 @@
+# Performance Metrics
+
+### Background
+
```

Comment:
you can remove the readme 
https://etwiki.sys.comcast.net/display/FIR/Ripple+Performance+Testing there is already a link with all details. just confirm if we have all the upto date info

Reply:
removed readme file